pulsar-client-cpp/lib/MessageId.cc

```diff
@@ -43,12 +52,59 @@ MessageId::MessageId(int64_t ledgerId, int64_t entryId)
      // consumer's partition is assigned to this partition
 }
 
+const MessageId& MessageId::earliest() {
+    static const BatchMessageId _earliest(-1, -1);
```

Comment:
Just curious what does `_var`convention mean I know that `var_` is for private members.

Reply:
I wasn't implying any convention. these are just locally defined static variables.